frontend/src/views/edit-project/RepresentationNavigation.module.css

```diff
@@ -17,6 +17,7 @@
 }
 
 .item {
+  columns: 2;
```

Comment:
Do not use CSS `column` layout, start with Flexbox and if really necessary to Grid

Reply:
Done